### DISCUSSION TITLE
need help dor capstone ML, scooty case

### DISCUSSION THREAD

**Discussion by tatapinandhika** [teamalgoritma/community#146] [Q&A] [answered]
saya mencoba untuk membuat model timeseries terhadap data scooty, tapi saya bingung saat ingin menentukan startnya karena terdapat data hours. Mohon bantuannya, terimakasih. 

<img width="1440" alt="Screen Shot 2022-06-21 at 15 46 00" src="https://user-images.githubusercontent.com/101564223/174745232-fcaf153f-2f17-45ab-ad53-cb8b5cf464bf.png">

**Comment by inytss** [comment, marked as answer]
Dear @tatapinandhika ,
Untuk membuat object time series menggunakan fungsi `ts()`, parameter `start` tidak perlu diikutsertakan dikarenakan parameter tersebut hanya akan memberikan index label saja pada data time seriesnya. Apabila tidak diikutsertakan pun tidak masalah dan tidak akan mengganggu dalam proses forecasting nantinya.
